src/h5web/test-utils.tsx

```diff
@@ -3,12 +3,26 @@ import { render, RenderResult } from '@testing-library/react';
 import App from './App';
 import MockProvider from './providers/mock/MockProvider';
 
-export const DOMAIN = 'source.h5';
-
 export function renderApp(): RenderResult {
   return render(
-    <MockProvider domain={DOMAIN}>
+    <MockProvider>
       <App />
     </MockProvider>
   );
 }
+
+/* eslint-disable no-console */
+export function prepareForConsoleError() {
+  const original = console.error;
+
+  const mock = jest.fn();
+  console.error = mock;
+
+  return {
+    consoleError: mock,
+    resetConsole: () => {
+      console.error = original;
+    },
+  };
+}
+/* eslint-enable no-console */
```

Comment:
When an error is thrown, React logs it into the console regardless of the presence of an `ErrorBoundary`. Since `setupTest` fails a test if a console error/warning was logged during its execution, I need a way to properly mock the console so it doesn't log anything and so I can check if it's been called or not.

Reply:
Perhaps a comment or a link describing this would be useful.